common/assets/scss/components/_whats-new-page.scss

```diff
@@ -0,0 +1,7 @@
+.section-heading {
+	margin-top: 40px;
+}
+
+.whats-new-section-title {
+	margin-bottom: 30px;
+}
```

Comment:
Can we use the [GOV.UK Design System spacing classes](https://design-system.service.gov.uk/styles/spacing/) here instead? 

I think `margin-top: 40px` would be `govuk-!-margin-top-7` and `margin-bottom: 30px` would be `govuk-!-margin-bottom-6`.